pages/content/amp-dev/documentation/index.html

```diff
@@ -0,0 +1,221 @@
+    link: /content/amp-dev/documentation/examples/index.html
+  - title: Templates
+    description: Don't want to start from scratch? Start with a ready-made design.
+    icon: template
+    link: /content/amp-dev/documentation/templates/index.html
+  - title: Tools
+    description: Begin Building for creation, design and development.
+    icon: wrench
+    link: /content/amp-dev/documentation/tools.html
+
+---
+{% do doc.amp_dependencies.add('amp-bind', '0.1') %}
+{% do doc.styles.addCssFile('/css/components/templates/overview.css') %}
+
+
+{% block main %}
+  <main class="ap--main ap-t-overview">
+    <div class="ap--container">
+      <div class="ap--content">
+
+        {% do doc.styles.addCssFile('/css/components/molecules/overview-title.css') %}
+        <section class="ap--section">
+          <div class="ap--overview-title">
+            <div class="ap-m-overview-title">
+              {% include '/views/partials/breadcrumbs.j2' %}
+              <h1>{{ doc.title }}</h1>
+              <p>Whether you are a publisher, e-commerce company, storyteller, advertiser or email sender, AMP makes it easy to create great experiences on the web. Use AMP to build:</p>
+            </div>
+          </div>
+        </section>
+
+        {% do doc.styles.addCssFile('/css/components/organisms/format-cards.css') %}
+        <section class="ap--section">
+          <div class="ap--format-cards">
+            <div class="ap-o-format-cards">
+              {% for card in doc.format_cards %}
+              <a href="{{g.doc(card.link.url, locale=doc.locale).url.path}}" class="ap-o-format-cards-card ap-o-format-cards-card-{{ card.type }}">
+                <div class="ap-o-format-cards-card-body">
+                  <div class="ap-a-ico ap-o-format-cards-card-icon">
+                    <svg>
+                      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#amp-{{ card.type }}"></use>
+                    </svg>
+                  </div>
+                  <h4 class="ap-o-format-cards-card-headline">{{ card.title }}</h4>
+                  <p class="ap-o-format-cards-card-copy">{{ card.description }}</p>
+                </div>
+                <div class="ap-o-format-cards-card-link ap-m-lnk">
+                  <div class="ap-a-ico ap-m-lnk-icon">
+                    {% do doc.icons.useIcon('icons/internal.svg') %}
+                    <svg>
+                      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use>
+                    </svg>
+                  </div>
+                  <span class="ap-m-lnk-text">{{ card.link.text }}</span>
+                </div>
+              </a>
+              {% endfor %}
+            </div>
+          </div>
+        </section>
+
+        {% do doc.styles.addCssFile('/css/components/organisms/resources-overview.css') %}
+        <amp-state id="chosenFormat" class="i-amphtml-layout-container i-amphtml-element" i-amphtml-layout="container" hidden="" aria-hidden="true">
```

Comment:
This has probably been copied from a hydrated site. Anything that looks like `i-amphtml-*` can't be part of the original source.